packages/backend/src/open_payments/receiver/service.test.ts

```diff
@@ -280,7 +289,11 @@ describe('Receiver Service', (): void => {
             .mockResolvedValueOnce(undefined)
         })
 
-        test('resolves incoming payment', async () => {
+        test.each`
+          rotate   | description
+          ${false} | ${''}
+          ${true}  | ${'- after rotating access token'}
+        `('resolves incoming payment $description', async ({ rotate }) => {
```

Comment:
I wonder if we can skip for `!existingGrant && rotate`

Reply:
I was wondering the same thing. I meant to ask you guys.

Reply:
I vote yes but I'm not sure how other than wrap the inside of the test with `if (!existingGrant && rotate) {`?